WordPress/src/main/java/org/wordpress/android/ui/jetpack/restore/RestoreViewModel.kt

```diff
@@ -374,7 +374,7 @@ class RestoreViewModel @Inject constructor(
         val currentIndex = wizardManager.currentStep
         val targetIndex = wizardManager.stepPosition(targetStep)
 
-        for (i in currentIndex downTo targetIndex) {
+        for (it in currentIndex downTo targetIndex) {
```

Comment:
Using `it` makes me uneasy. Do you think the following would be better, so we can use `_` instead?

```suggestion
        (currentIndex downTo targetIndex).forEach { _ ->
```